lib/managers/folders.js

```diff
@@ -8,7 +8,9 @@
 // ------------------------------------------------------------------------------
 // Requirements
 // ------------------------------------------------------------------------------
-var urlPath = require('../util/url-path');
+var urlPath = require('../util/url-path'),
+	errors = require('../util/errors'),
+	httpStatusCodes = require('http-status');
```

Comment:
You don't need the `errors` or `httpStatusCodes` dependencies anymore; they can be safely removed.